### PR TITLE
APIS-7266 - Update api description overrides

### PR DESCRIPTION
**Pull request by Tobias1087**

### CODE REVIEW

File: resources/api-description-overrides.json [hunk 1]
```diff
@@ -1,6 +1,582 @@
...
+  {
+    "description": "The Telephony Identity Verification API",
+    "identifier": "telephony-identity-verification-api"
+  },
+  {
+    "description": "The Telephony Identity Verification Test API",
+    "identifier": "telephony-identity-verification-test-api"
+  },
+  {
+    "description": "A API for Time To Pay Proxy.",
```
Comment (worthydolt) [inline resources/api-description-overrides.json:551]: Admittedly this matches the sheet, but "A API"?

File: resources/api-description-overrides.json [hunk 2]
```diff
@@ -1,6 +1,582 @@
...
+  {
+    "description": "A 'hello world' example of an API on the HMRC API Developer Hub.",
+    "identifier": "api-example-microservice"
+  },
+  {
+    "description": "Allows for retrieval of notifications",
+    "identifier": "api-notification-pull"
+  },
+  {
+    "description": "The Create Test User API is responsible for creating test users on the external test environment.",
```
Comment (johnsgp) [inline resources/api-description-overrides.json:23]: This should be "sandbox" not "external test". This is public facing where it's referred to as "sandbox".

File: resources/api-description-overrides.json [hunk 3]
```diff
@@ -1,6 +1,582 @@
...
+  {
+    "description": "Determines if a supplied NINO and user details match data according to records stored by HMRC",
+    "identifier": "authenticator"
+  },
+  {
+    "description": "Validation, verification and insights into Bank Accounts - provided by the Customer Insight Platform",
+    "identifier": "bank-account-gateway"
+  },
+  {
+    "description": "Verify that a personal or business bank account exists",
```
Comment (johnsgp) [inline resources/api-description-overrides.json:39]: Why do some descriptions end with `.` and others not? It feels untidy.

File: resources/api-description-overrides.json [hunk 4]
```diff
@@ -1,6 +1,582 @@
...
+  {
+    "description": "Allows for retrieval of EORI numbers",
+    "identifier": "check-eori-number-api"
+  },
+  {
+    "description": "Get insights for a bank account",
+    "identifier": "cip-bank-account-insights"
+  },
+  {
+    "description": "An API for providing Construction industry scheme data",
```
Comment (johnsgp) [inline resources/api-description-overrides.json:79]: `Construction Industry Scheme` is a thing - should it all be capitalised?

File: resources/api-description-overrides.json [hunk 5]
```diff
@@ -1,6 +1,582 @@
...
+  {
+    "description": "The public API to allow traders to submit and receive Common Transit Convention (IE) messages",
+    "identifier": "common-transit-convention-traders"
+  },
+  {
+    "description": "The public API to allow 3rd party testers to inject Common Transit Convention (IE) messages as if they were coming from NCTS",
+    "identifier": "common-transit-convention-traders-test-support"
+  },
+  {
+    "description": "Api to manage compliance cases in CaseFlow",
```
Comment (johnsgp) [inline resources/api-description-overrides.json:95]: `API` not `Api`

**Review by mattclark-zerogravit — APPROVED** [review]
👍

**Pull request merged**